options/locale/locale_en-US.ini

```diff
@@ -2432,7 +2432,7 @@ auths.allowed_domains = Allowed Domains
 auths.allowed_domains_helper = Leave empty to allow all domains. Separate multiple domains with a comma (',').
 auths.skip_tls_verify = Skip TLS Verify
 auths.force_smtps = Force SMTPS
-auths.force_smtps_helper = By default SMTPS will be used for port 465, set this to use smtps on other ports, otherwise STARTTLS is used if supported.
+auths.force_smtps_helper = By default, SMTPS uses port 465. Set this to force SMPTS on other ports as well. Otherwise, STARTTLS will be used for these ports if it is supported.
```

Comment:
```suggestion
auths.force_smtps_helper = SMTPS is always used on port 465. Set this to force SMTPS on other ports. (Otherwise STARTTLS will be used on other ports if it is supported by the host.)
```